core/embed/sec/secret/unix/certs/gen.sh

```
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)